README.md

```diff
@@ -44,6 +44,26 @@ containers, the containers run `/bin/true`:
 | 100 /bin/true (no network namespace)  | 0:05.70        | 0:10.95 | -47.9% |
 | 100 /bin/true (new network namespace) | 0:06.16        | 0:11.17 | -44.8%  |
 
+crun requires less resources, so it is also possible to set stricter
```

Comment:
very small nit: gramatically it should be
`crun requires fewer resources`

Reply:
thanks for pointing it out!  I've pushed a new version.